### PR TITLE
hotfix: Fix Backups Drawer, Region Select Order, Cloud Manager `v1.98.1`

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Fix Backups Drawer by manually copying relevant changes from https://github.com/linode/manager/pull/9421
- Fixes Region Select Order

## Preview 📷

| Before  | After   |
| ------- | ------- |
| Screenshot 2023-07-28 at 1 53 37 PM | Screenshot 2023-07-28 at 1 53 10 PM |
| Screenshot 2023-07-28 at 1 58 13 PM|Screenshot 2023-07-28 at 1 58 31 PM |

## How to test 🧪
- Test the backups drawer
  - You need some linodes on your account without backups 
  - If you don't see the backups drawer banner, go to `http://localhost:3000/profile/settings?preferenceEditor=true` and clear your preferences by setting the value to `{}`
- Verify region select order is ...

### CODE REVIEW

**Review by jaalah-akamai — APPROVED** [review]
Fixes look good

File: packages/manager/src/components/EnhancedSelect/variants/RegionSelect/RegionSelect.test.tsx [hunk 1]
```diff
@@ -10,11 +10,11 @@ describe('Region Select helper functions', () => {
       const groupedRegions = getRegionOptions(regions);
       const [r1, r2, r3, r4, r5] = groupedRegions;
```
Comment (mjac0bs) [inline packages/manager/src/components/EnhancedSelect/variants/RegionSelect/RegionSelect.test.tsx:11]: Maybe not the most readable of naming schemes. 😄

**Review by mjac0bs — APPROVED** [review]
The region dropdown order and the backups drawer linodes are behaving as expected now; I also just ran the fixed unit test locally to confirm. ✅

File: packages/manager/src/components/EnhancedSelect/variants/RegionSelect/RegionSelect.test.tsx (hunk 1 above)
Comment (bnussman-akamai) [reply packages/manager/src/components/EnhancedSelect/variants/RegionSelect/RegionSelect.test.tsx:11]: yeah this is not a great test. I will make a ticket to improve this.

**Review by jaalah-akamai — APPROVED** [review]

**Pull request merged**